src/java/org/apache/cassandra/service/accord/AccordCache.java

```diff
@@ -1212,7 +1212,7 @@ public Object fullShrink(TxnId txnId, Command value)
 
             try
             {
-                return AccordJournal.asSerializedChange(null, value, current_version);
+                return AccordJournal.asSerializedChange(null, value, Version.LATEST);
```

Comment:
TODO: is this in-memeory or a disk cache?  if disk we shouldn't use `latest` but instead a downgrade safe version.